concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -101,40 +94,87 @@ def report_computed_task(timestamp = None, subtask_id = None, task_to_compute =
         )
 
 
+def create_task(task_to_compute_timestamp, task_id, compute_task_def_deadline):
+    return task_to_compute(
+        timestamp=task_to_compute_timestamp,
+        compute_task_def=compute_task_def(
+            task_id=task_id,
+            subtask_id=task_id,
+            deadline=compute_task_def_deadline,
+        )
+    )
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
+                               ["concent_messaging_time",
+                                "subtask_verification_time",
+                                "force_acceptance_time",
+                                "token_expiration_time"])
+
+
+def get_protocol_constants(cluster_url):
+    url = f"{cluster_url}/api/v1/protocol-constants/"
```

Comment:
As I generally like this new feature, I would suggest to hold with it, I know that on some our environments we still use Python3.5, and that would block using whole script on this version.

Reply:
according to information from @cameel, we will need to upgrade to python3.6 before the release, so it should not be the problem ;)

Reply:
Ok, great.

Reply:
Python 3.6 is now on the cluster. You can safely assume that it's at least 3.6.